.github/workflows/celo-monorepo.yml

```diff
@@ -45,7 +45,10 @@ jobs:
       # Adding a initial comma so ',<path>' matches also for the first file
       all_modified_files: ',${{ steps.changed-files.outputs.all_modified_files }}'
     # runs-on: ubuntu-latest
-    runs-on: ['self-hosted', 'monorepo-node18']
+    runs-on: ['self-hosted', 'org', '8-cpu']
+    container:
+      # Update image at https://github.com/celo-org/infrastructure/blob/master/terraform/root-modules/gcp/integration-tests-gke/files/github-arc/Dockerfiles/Dockerfile-monorepo
+      image: us-west1-docker.pkg.dev/devopsre/actions-runner-controller/celo-monorepo:node18
```

Comment:
is it possible to set the image name / tag in the `env:` section? and make future upgrade to (e.g.) `celo-monorepo:node19` simply changing the value of the image ref in `env` ?

Reply:
🤔  It may be 🤔

Reply:
sadness...
![image](https://github.com/celo-org/celo-monorepo/assets/5635989/d89c10f9-9cf2-4c82-9308-f5089aa4662f)

Reply:
oof yeah there's a few places the templating doesn't work :(